client/app/hosting/database/hosting-database.service.js

```diff
@@ -92,14 +92,33 @@ angular
          * Get dump details
          * @param {string} serviceName
          * @param {string} name
-         * @param {string} dump
+         * @param {string} dumpId
          */
-        getDump (serviceName, name, dump) {
-            return this.OvhHttp.get(`/hosting/web/${serviceName}/database/${name}/dump/${dump}`, {
+        getDump (serviceName, name, dumpId) {
+            return this.OvhHttp.get(`/hosting/web/${serviceName}/database/${name}/dump/${dumpId}`, {
                 rootPath: "apiv6"
+            }).then((dump) => {
+                dump.snapshotDate = this.constructor.getSnapshotDateOfDump(dump);
+                return dump;
             });
         }
 
+        /**
+         * Get snapshot date of database dump
+         * @param {Object} dump
+         */
+        static getSnapshotDateOfDump (dump) {
+            if (!dump.creationDate) { return undefined; }
```

Comment:
I'd also rather have you check if `dump.creationDate` is a string and is not empty:
`_(dump.creationDate).isString() && !_(dump.creationDate).isEmpty()`

Reply:
I could also check every String return by the Api, move on and stop making me using lodash for everything